### DISCUSSION TITLE
<Feature Request> Name pronounication recording

### DISCUSSION THREAD

**Discussion by CBID2** [community/community#38891] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi everyone! I was randomly scrolling through GitHub, and an idea came to me. How about we add a feature where users can insert a recording of themselves saying their names? As a person who's last name often gets mistaken for a country, having this feature would help recruiters interact with users from diverse backgrounds, thus creating a positive impact on said individual. Here's a source that gives more detail about this issue: [Why Pronouncing Names Correctly Is More Than Common Courtesy](https://www.npr.org/2021/03/12/976385244/why-pronouncing-names-correctly-is-more-than-common-courtesy)

**Comment by jfullstackdev** [comment]
I think that's a valid request, unfortunately GitHub is triaging so much issues and feature requests too

**Reply by CBID2** [reply]
Thanks @jdevstatic! Hopefully this feature will be created. I mean LinkedIn does it.
